src/main/java/com/atlauncher/App.java

```diff
@@ -498,6 +499,47 @@ private static void checkIfNeedToUpdateBundledJre() {
         }
     }
 
+    private static void checkIfUsingOutdatedJava() {
+        if (ConfigManager.getConfigItem("outdatedJavaPrompt.enabled", false) == true
```

Comment:
Why are you adding `== true`? `getConfigItem` should work on generics, and the return will be a boolean due to the default value being boolean.

Reply:
Hmm yeah not sure, let me follow up on that make sure things still work

Reply:
@Doomsdayrs yeah I think this is the case as the generic type is `Boolean` so it could technically come out as null, so needs the equality check.

ConfigManager is shit, I don't like how it is currently, I want to rewrite it since I made it but it's a dynamic object so had no better idea to make sure backwards compat was kept. If I typed it and just GSONed it into an object, would've caused issues with older clients that haven't updated yet.